### PR TITLE
Extract formulas

### PR DESCRIPTION
**Pull request by alexeymorozov**
See #124

### CODE REVIEW

File: lib/fontist/utils/git_requirement.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+module Fontist
```
Comment (ronaldtse) [inline lib/fontist/utils/git_requirement.rb:1]: Can we use https://github.com/ruby-git/ruby-git here?

**Review by ronaldtse — CHANGES_REQUESTED** [review]
How do you expect users to update Formulas? e.g. a CLI command `fontist update`?

File: lib/fontist/utils/git_requirement.rb (hunk 1 above)
Comment (alexeymorozov) [reply lib/fontist/utils/git_requirement.rb:1]: Let's see :)

Comment (alexeymorozov) [reply lib/fontist/utils/git_requirement.rb:3]: Done. Kept the GitRequirement class for more human-readable error if there is no git in a system.

**Comment by alexeymorozov** [comment]
> > How do you expect users to update Formulas? e.g. a CLI command `fontist update`?
> 
> Currently the formulas repo is pulled on each load of the library. Do you see any downsides of this implementation?

Decided to build the update command in a new PR

**Pull request merged**